products/bash/operator_run/setup-demos.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2020. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#******************************************************************************
+
+#******************************************************************************
+
+CURRENT_DIR=$(dirname $0)
+
+while getopts "i:o:" opt; do
+  case ${opt} in
+  i)
+    in_file="$OPTARG"
+    ;;
+  o)
+    out_file="$OPTARG"
+    ;;
+  \?)
+    usage
+    exit
+    ;;
+  esac
+done
+
+echo "${in_file}"
+echo "${out_file}"
+
+mkdir /tmp/git-repo 
+git clone --branch main https://github.com/IBM/cp4i-deployment-samples.git /tmp/git-repo
```

Comment:
I would prefer this worked the same way as the catalog install.sh, using wget and unzip. Does this work from inside the operator? I suspect git is not part of the operator image and wget/unzip are.